app/components/DisplayWinner.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 
+// this is a little small to be its own component
 const DisplayWinner = ({ winner }) => {
```

Comment:
It also has a specific purpose and not reusable - probably another reason to not make it a component? What criteria do you use when deciding to make something a component?

Reply:
purely stylistic. if you're more comfortable leaving it, it's no big deal. 
I like to have small bits of jsx without much functionality to sometimes be better when left in the larger component.